src/parseTools.js

```diff
@@ -1040,14 +1040,18 @@ function makeHEAPView(which, start, end) {
   return 'HEAP' + which + '.subarray((' + start + ')' + mod + ',(' + end + ')' + mod + ')';
 }
 
-function makeDynCall(sig) {
+function exportedAsmFunc(func) {
```

Comment:
"exported" is maybe not clear enough. The meaning is "if dynamically linking, this must be handled in a special way". How about `makePotentiallyLinkedCall` and add a comment, `when dynamically linking, some things like dynCalls may not exist in one module and be provided by a linked module, so they must be accessed indirectly using Module`?

Reply:
It is odd to highlight `makePotentiallyLinkedCall` here. It would be the same as `makeGetValue()` was renamed to `makePotentiallySafeHeapGetValue()`, or `makeDynCall()` was renamed to `makePotentiallyLinkedDynCall()`, i.e. it is highlighting one possible special configuration of the option that may need to occur, rather the general intent of that function (`makeGetValue()` provides a access to heap, `makeDynCall()` produces invocation to a function ptr, `exportedAsmFunc()` accesses a function that was exported from asm module)

I mean, I am indifferent to this, I can do the rename, but it feels weird to highlight one of the options that could as well be an internal implementation detail?

Reply:
I see what you mean, good point. Maybe just add the comment then?